native/src/components/Header.tsx

```diff
@@ -122,13 +142,42 @@ const Header = ({
     }
   }
 
+  const getCategorySlug = (path?: string): string | undefined => {
+    if (!path) {
+      return undefined
+    }
+    return getSlugFromPath(path)
+  }
+
+  const getSlugForRoute = (): string | undefined => {
+    switch (route.name) {
+      case EVENTS_ROUTE:
+        return (route.params as RoutesParamsType[EventsRouteType]).slug
+
+      case POIS_ROUTE:
+        return (route.params as RoutesParamsType[PoisRouteType]).slug
+
+      case CATEGORIES_ROUTE:
+        return getCategorySlug((route.params as RoutesParamsType[CategoriesRouteType]).path)
+
+      case SPRUNGBRETT_OFFER_ROUTE:
+        return SPRUNGBRETT_OFFER_ROUTE
+
+      case DISCLAIMER_ROUTE:
+        return (data as UseLoadExtraCityContentData<PageModel>).extra.slug
```

Comment:
I think this should actually always be `disclaimer` so you should be able to just change this to
```suggestion
        return DISCLAIMER_ROUTE
```